CSS/styles.css

```diff
@@ -1,71 +1,79 @@
 :root {
   --almost-black: #252121;
+  --GhostWhite: #F8F8FF;
 }
 
-div{
+.aboutMe {
+  float: left;
+  display: inline-block;
+}
+
+.bottomContainer {
+  background-color: yellow;
+}
+
+.container-h1 {
+  font-size: 2rem;
+  top: 150px;
+  right: 110px;
+}
+
+.container-h2 {
+  font-size: 1.5rem;
+}
+
+.container-p {
+  font-size: 1rem;
+  top: 210px;
+  right: 30px;
+}
+
+.container-image {
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+}
+
+div {
+  font-family: 'PT Mono', monospace;
+  text-align: center;
+}
+
+.mainDiv{
   height: 500px;
   margin: 5%;
   padding: 70px;
-  font-family: Courier New, monospace;
-  text-align: center;
 }
-.noMargin{
-  margin: 0;
-}
-.topContainer{
-  background-color: var(--almost-black);
-  color: white;
+
+.subDiv{
+  width: 50%;
 }
-.middleContainer{
+
+.middleContainer {
   color: var(--almost-black);
-  border: solid 20px;
+  border: solid 10px;
   padding: 0;
-  position: relative;
   left: 0;
 }
-.bottomContainer{
-  background-color: yellow;
-}
-.geoImage{
-  position: relative;
-  height: 100px;
-}
-.geoImage1{
-  top: 0;
-  background-color: red;
+
+.noMargin {
+  margin: 0;
 }
-.geoImage2{
-  top: 200px;
-  background-color: blue;
+
+.positionAbsolute {
+  position: absolute;
 }
-.geoImage3{
-  bottom: 0;
-  background-color: green;
-/*.bubbleSize{
-  width: 100%;
-  height: 100%;*/
+
+.positionRelative{

```

Comment:
er mist een spatie tussen `positionRelative` en `{`. dit is typisch iets wat een (automatische) code formatter voor je zou oplossen. dus of door middel van een key shortcut, of een linting tool. zie ook mijn recente berichten in Slack

Reply:
done! zag ook gelijk dat ik het ook bij `subDiv` ook was vergeten